students_generator.py

```diff
@@ -0,0 +1,62 @@
+import random
+
+
+def generate_student_name():
+    first_names = ["John", "Jane", "Emily", "Chris", "Alex", "Michael", "Emma", "Olivia",

```

Comment:
This should be moved to consts and defined before all funcs (at the top of the file)